src/Robin/Ntlm/Message/NtlmV1AuthenticateMessageEncoder.php

```diff
@@ -229,6 +241,10 @@ public function encode(
     /**
      * Calculates the LM response.
      *
+     * TODO: Remove this method as it's no longer necessary.
```

Comment:
I was thinking of it, but I'd have to bump another version. I can just clean these up later in a more universal BC break release.